src/arrayMethodSort.js

```diff
@@ -5,7 +5,29 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let operationCount = -1;
+
+    while (operationCount !== 0) {
+      operationCount = 0;
```

Comment:
maybe something like `isSwapped`, doesn't matter how many swaps did, enough to know that swap is did